spec/features/posts_feature_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+require 'web_helper'
+
+feature 'posts' do
+  context 'no posts have been added' do
+    scenario 'should display a prompt to add a post' do
+      visit '/posts'
+      expect(page).to have_content 'No posts yet'
+      expect(page).to have_link 'Add a post'
+    end
+  end
+
+  context 'posts have been added' do
```

Comment:
This would be cleaner:

```ruby
context 'posts have been added' do

    let!(:post) { Post.create(caption: 'Here is a test post') }

    scenario 'display posts' do
      visit '/posts'
      expect(page).to have_content(post.caption)
      expect(page).not_to have_content('No posts yet')
    end
  end
```